### ISSUE TITLE
prototext files

### ISSUE THREAD

**Issue by cjsure** [sciencefans/Focal-Loss#7]
can you provide train_test.prototext and solver.prototext file?

**Comment by liuyuisanai** [comment]
Please refer to Readme.
> Using it as the way you use cross entropy loss layer.

For softmax-form focal loss, please refer to this repo: https://github.com/zimenglan-sysu-512/Focal-Loss
